### PR TITLE
chore(opa): add 0.67.1

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
See: https://github.com/open-policy-agent/opa/releases/tag/v0.67.1

```[tasklist]
### Dependencies
- [ ] Vector product update
```

### CODE REVIEW

**Comment by NickLarsenNZ** [comment]
Should we now remove 0.61.0 since it was deprecated in the last release?

**Review by lfrancke — APPROVED** [review]

File: opa/versions.py
Comment (NickLarsenNZ) [inline opa/versions.py:4]: I think we need to update Vector first (if it gets updated)

Comment (sbernauer) [reply opa/versions.py:4]: IMHO it's fine to first bump OPA and vector afterwards

**Review by sbernauer — APPROVED** [review]

File: opa/versions.py
Comment (NickLarsenNZ) [reply opa/versions.py:4]: Oh yeah, it will just be a new nightly to replace the one created by this.

_I didn't see your review, and updated the description to add Vector as a dependency. Will leave it there until I come back to this (maybe tomorrow)._

Comment (sbernauer) [reply opa/versions.py:4]: Honestly, I didn't fully get it. I leave it to you, I'm happy to either merge or wait :)

Comment (NickLarsenNZ) [reply opa/versions.py:4]: I was just agreeing with you that it doesn't matter if they're done separately (this early), because:

1. Once this is merged, it will build the dev image with old Vector
2. Once Vector is updated, a new nightly will be built since the dev workflow would trigger it:
   https://github.com/stackabletech/docker-images/blob/056274f976de333caeb9ac9be4b3b8686109131f/.github/workflows/dev_opa.yaml#L18

The last line in _italics_ was just meaning I will wait because it was late and I didn't want to rush a merge. Now I'm happy to merge it and let it rebuild once Vector gets updated.

**Comment by NickLarsenNZ** [comment]
I ran `./upload_new_opa_version.sh 0.67.1 ...`

**Pull request merged**
